aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -342,6 +349,63 @@ module aptos_framework::account {
         rotation_capability
     }
 
+    /// Cffer signer capability to another address
```

Comment:
Typo: Cffer should be Offer

Reply:
This doesn't clarify who's offering to whom. A more instructive comment would be:

 > Offers the capability to sign on behalf of `account` to the account at address `recipient_address`.